{
  "cve_id": "CVE-2022-31047",
  "timestamp": "2025-01-10 07:49:08",
  "phases": {
    "primary": {
      "completed": true,
      "timestamp": "2025-01-08 10:43:26"
    },
    "secondary": {
      "completed": true,
      "timestamp": "2025-01-09 11:55:02"
    },
    "extraction": {
      "completed": true,
      "timestamp": "2025-01-10 04:51:50"
    }
  },
  "urls": [
    {
      "url": "https://github.com/TYPO3/typo3/commit/c93ea692e7dfef03b7c50fe5437487545bee4d6a",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-09 20:44:49"
    },
    {
      "url": "https://github.com/TYPO3/typo3/security/advisories/GHSA-fh99-4pgr-8j99",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-09 20:44:49"
    },
    {
      "url": "https://typo3.org/security/advisory/typo3-core-sa-2022-002",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-09 20:44:49"
    }
  ]
}